guides/common/modules/proc_deleting-openscap-reports.adoc

```diff
@@ -13,56 +13,54 @@ Example request:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
-$ curl --user _My_User_Name_:__My_Password__ \
+$ curl \
+--user _My_User_Name_:__My_Password__ \
 https://_{foreman-example-com}_/api/v2/compliance/arf_reports/ | python3 -m json.tool
 ----
 +
 Example response:
 +
-[options="nowrap", subs="+quotes,attributes"]
+[source, none, options="nowrap", subs="+quotes,attributes"]
 ----
-  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
```

Comment:
I have removed this part of the stdout of `curl`. Should we always add `--silent` and `--show-error` to curl commands?

Reply:
It's okay to remove the stdout part. I'm not sure about the arguments. My personal preference is to reduce arguments in examples to a bare minimum, I wouldn't force any on users.